README.md

```diff
@@ -4,165 +4,172 @@ The ATAT Web API is the internal application programming interface that supports
+1. Navigate to a directory containing a `main.tf` file.
+2. Prepare a file `terraform.tfvars` from file `terraform.tfvars.template` in the same directory (if available)
+3. Ensure you are logged in to Azure with `az login`
+4. Set your desired subscription with `az account set -s <subscription-id>`.  You can view the selected subscription with `az account show` 
 
-#### Terraform Init
+#### Terraform Steps
 
-You will need the name of the storage account where state will be stored as well as the
-resource group that the storage account is in. Those will be called `$AZURE_STORAGE_ACCOUNT` and
-`$AZURE_STORAGE_RESOURCE_GROUP` respectively in any subsequent steps.
+##### Initialization
+You will need the name of a storage account where Terraform [state](https://www.terraform.io/docs/language/state/index.html) will be stored as well as the [resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#what-is-a-resource-group) that contains it. Those will be called `$AZURE_STORAGE_ACCOUNT` and `$AZURE_STORAGE_RESOURCE_GROUP` respectively in any subsequent steps.
 
 ```
 terraform init -backend-config=resource_group_name=$AZURE_STORAGE_RESOURCE_GROUP -backend-config=storage_account_name=$AZURE_STORAGE_ACCOUNT
 ```
 
-#### Retrieve .tfstate from Azure
+##### Select a workspace
 
-Select the correct workspace inside the container on Azure
+Select the desired workspace.
+Terraform state is stored by default in a file named _terraform.tfstate_.  In this case it is stored remotely inside the storage container on Azure.
 
 ```
-terraform workspace select <name of workspace>
+terraform workspace select <your-workspace-name>
 ```
 
-You are now connected to the remote .tfstate file on Azure.
+You are now connected to the remote .tfstate file.
 
-#### Plan and Apply
+##### Plan and Apply
 
 To create an execution plan, run:
 
 ```
 terraform plan
 ```
 
-If Terraform detects that no changes are needed to resource instances or to root module output values, terraform plan will report that no actions need to be taken.
+If Terraform detects that no changes are necessary, `plan` will report that no actions need to be taken.
 
 To apply your changes, run:
 
 ```
 terraform apply
 ```
 
-To destroy the deployed shared_infrastructure run:
+To destroy the deployed shared infrastructure run:
 
 ```
 terraform destroy
 ```
 
-#### Create Azure Service Principal and set environment variables
+### Deploy Azure Function Apps
+---
+[Serverless Framework](https://www.serverless.com/) will be used to deploy Azure Function Apps.
 
-Follow steps at [Creating a Service Principal](https://www.serverless.com/framework/docs/providers/azure/guide/quick-start#creating-a-service-principal) at serverless.com
+The following is a list of `serverless.yml` files and their locations which can be used with Serverless Framework to deploy Function Apps:
+* `./provisioning/serverless.yml`
+* `./poc/random_quote/serverless.yml`
```

Comment:
Ditto to above comment, can remove PoC items.

Reply:
Affirmative.